dmgroups.tex

```diff
@@ -135,29 +135,47 @@ \subsection{DM Leadership Team \label{sect:dmlt}}
 
 \begin{itemize}
-\item Charter/purpose
-	\begin{itemize}
-	\item Ensure that the DM Technical Baseline (LDM-xxx) documents are baselined and subsequently changed only when necessary and according to LSST and DM configuration control processes
-	\end{itemize}
-\item Membership
+\item Responsibilities:
 	\begin{itemize}
-	\item Chaired by the Systems Engineer (\secref{role:sysengineer}).
-	\item The DMCCB has the same membership as the Systems Engineering Team (\secref{sect:sysengt}).
-	\item For on-line virtual meetings, if a consensus or quorum or is not reached within one week, the DM Project Manager will make a unilateral decision
+	\item Determine when deliverables (controlled documents and software) are ready to be baselined (placed under configuration controlled status) or released. This include LDM series documents.
+	\item Review and approve/reject proposed changes to baselined items
+        \item Review and approves all RFCs prior to 'Adoption'.
+	\item Monitor and approve  DM software releases
+        \item Monitor DM Jira issues status.
+	\item Ensure that the DM  follows  LSST and DM configuration control processes.
 	\end{itemize}
-\item Responsibilities
+\item Membership:
 	\begin{itemize}
-	\item Determines when specification and deliverables are of sufficient maturity and quality to be baselined (placed under configuration controlled status) or released.
-	\item Reviews and approves/rejects proposed changes to baselined items
+        \item Core members:
+                \begin{itemize}
+                \item DM Project Manager
+                \item DM Subsystem Scientist
+	        \item Chaired by the Systems Engineer (\secref{role:sysengineer}).
+                \item Operations Architect
+                \item Software Architect
+                \item DM Configuration and Release Manager (secretary)
+                \end{itemize}
+        \item Optional members are involved when topics to discuss are relevant to their areas of expertise:
+                \begin{itemize}
+                \item DMPM Deputy, when DMPM is not available
+                \item DM Deputy Subsystem Scientist, when DM Subsystem Scientist is not available
+                \item Pipeline Scientist
+                \item Science Pipelines / Interface Scientist
+                \item T/CAMs, who can delegate to their deputies
+                \end{itemize}
+	\item For on-line virtual meetings, if a consensus or quorum is not reached within one week, the DM Project Manager will make a unilateral decision
+        \item DMPM can also make unilateral decisions in cases of urgency. In that case DMCCB will assess the change a \textit{posteriori}.
 	\end{itemize}
 \end{itemize}
 
+The DM-CCB will meet, physically or virtually, every week for 30 minutes. Agenda will be available beforehand.
```

Comment:
“An agenda”. At least 30 minutes? At most 30 minutes? Presumably the meeting isn't required to last for 30 minutes if there's nothing to say. Can discussions go on longer than 30 minutes?

Reply:
The idea is to keep the meeting short. It can last less than 30 minutes, but not more.